WalletWasabi/Microservices/ProcessBridge.cs

```diff
@@ -77,18 +67,21 @@ public Process Start(string arguments, bool openConsole)
 			}
 		}
 
-		public async Task<(string response, int exitCode)> SendCommandAsync(string arguments, bool openConsole, CancellationToken cancel)
+		public virtual async Task<(string response, int exitCode)> SendCommandAsync(string arguments, bool openConsole, CancellationToken cancel)
 		{
-			int exitCode;
-
 			using var process = Start(arguments, openConsole);
+			Send(process.StandardInput);
 			await process.WaitForExitAsync(cancel).ConfigureAwait(false);
 
-			exitCode = process.ExitCode;
+			var exitCode = process.ExitCode;
```

Comment:
Why is the `exitCode` variable necessary?

Reply:
It is not necessary.